anndata/_core/anndata.py

```diff
@@ -1480,6 +1480,48 @@ def _mutated_copy(self, **kwargs):
             new["raw"] = self.raw.copy()
         return AnnData(**new)
 
+    def _to_memory_copy(self):
+        """Creates a new adata that has the computed results of the previous adata.
+        This is for the unbacked case
+        """
+        new = {}
+
+        if self.obsm is not None:
+            new["obsm"] = AxisArrays(
+                self, 0, vals={k: to_memory(v) for k, v in self.obsm.items()}
+            )
```

Comment:
Why are these `AxisArrays` and not just `dict`s?

Reply:
fixed